version.tf

```
@@ -1,3 +1,7 @@
 terraform {
   required_version = "~> 0.12.0"
+
+  required_providers {
+    github = "2.9.0"
```

Shouldn't this be more loosely pinned?

```suggestion
    github = "~> 2.9"
```

or perhaps:

```suggestion
    github = ">= 2.9.0"
```

I would go with the former to ensure the module continues to work as expected when 3.x of the provider is available, but leaving that up to you.

---

Yes, you're correct. Stupid... Fixed!